.github/workflows/diffs.yml

```diff
@@ -0,0 +1,346 @@
+---
+name: create-diffs
+on:
+    pull_request:
+        types: ["opened", "synchronize"]
+jobs:
+    tpgb:
+        runs-on: ubuntu-latest
```

Comment:
Can we use an image where we've set up Go / Ruby / all the boilerplate already? We repeat that stuff a few times.

Reply:
Absolutely doable, but this is inexpensive (the GH Actions container is optimized for this) and doesn't require us to maintain our own container.  This will save us a lot of maintenance burden.  Discussed offline and sounds like this is ok.